internal/pkg/scaffold/helm/role_test.go

```diff
@@ -0,0 +1,195 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helm_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/operator-framework/operator-sdk/internal/pkg/scaffold/helm"
```

Comment:
Nit: leave newline for import grouping.
```suggestion
	"github.com/operator-framework/operator-sdk/internal/pkg/scaffold/helm"
	
```